src/Grpc.AspNetCore.Server/GrpcBindingOptions.cs

```diff
@@ -0,0 +1,38 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+using System;
+using Grpc.AspNetCore.Server.Internal;
+using Grpc.Core;
+
+namespace Grpc.AspNetCore.Server
+{
+    /// <summary>
+    /// Options used to configure the binding of a gRPC service.
+    /// </summary>
+    public class GrpcBindingOptions<TService>
```

Comment:
I think I'm happy with the GrpcBindingOptions for now, if the assumption is that we will revisit the API for mapping/binding grpc services and methods in the future before we treat this API as stable (I think we should do an API review at some point as proposed in https://github.com/grpc/grpc-dotnet/issues/111#issuecomment-465504107).